doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -863,6 +863,10 @@ Other libraries
 
   * Fixed issue where the adp536x driver was included in the immutable bootloader on Thingy:91 when :kconfig:option:`CONFIG_SECURE_BOOT` was enabled.
 
+* :ref:`lib_date_time` library:
+
+  * Added date_time_now_local() to API.
```

Comment:
```suggestion
  * Added the :c:func:`date_time_now_local()` function to the API.
```

Reply:
@peknis
Should this be without `()`? That's how I've seen it usually when using `:c:func:` referencing.

Reply:
Good point, probably without, yes.